src/util/Numerical.js

```diff
@@ -159,7 +159,13 @@ var Numerical = new function() {
          * http://whizkidtech.redprince.net/bezier/circle/kappa/
          */
         KAPPA: 4 * (sqrt(2) - 1) / 3,
-
+        /**
+         * Amount to multiply radius of blur by to account for error
+         * in gaussian blur kernel filter approximation
+         *
+         * https://dbaron.org/log/20110225-blur-radius
+         */
+        GAUSSIAN_BLUR_COEFFICIENT: 1.4,
```

Comment:
Could you explain how you ended up with `1.4` from the cited document? I don't see any mention of 1.4 in there?

Reply:
Could it actually be `Math.SQRT2 ( === Math.sqrt(2))`?

Reply:
@lehni apologies I seem to have missed this.

1.4 was just in my testing what I found to be sufficiently large to cover extreme cases. 

I don't know how to calculate the actual error but this number doesn't need to precise, it's a tradeoff between precision and speed where the larger number is more accomadating but slower as it creates a larger in memory canvas.